### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by janekolszak**
- Update golangci-lint
- Fix linter errors

### CODE REVIEW

File: app/export.go [hunk 1]
```diff
@@ -175,9 +175,12 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 		}
 		return false
 	})
+	if err != nil {
+		panic(fmt.Errorf("error when iterating through redelegations: %w", err))
```
Comment (szynwelski) [inline app/export.go:179]: Maybe instead of using `fmt.Errorf` use `errors.Wrap` from "cosmossdk.io/errors"

File: app/ibc.go [hunk 2]
```diff
@@ -53,9 +51,9 @@ func (app *App) registerIBCModules() {
 	}
 
 	// register the key tables for legacy param subspaces
-	keyTable := ibcclienttypes.ParamKeyTable()
-	keyTable.RegisterParamSet(&ibcconnectiontypes.Params{})
-	app.ParamsKeeper.Subspace(ibcexported.ModuleName).WithKeyTable(keyTable)
+	// keyTable := ibcclienttypes.ParamKeyTable()
```
Comment (szynwelski) [inline app/ibc.go:54]: Maybe delete?

File: app/export.go [hunk 3]
```diff
@@ -223,12 +232,18 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
...
-			app.SlashingKeeper.SetValidatorSigningInfo(ctx, addr, info)
+			err := app.SlashingKeeper.SetValidatorSigningInfo(ctx, addr, info)
+			if err != nil {
+				panic(err)
+			}
 			return false
 		},
 	)
+	if err != nil {
+		log.Fatal(err)
```
Comment (szynwelski) [inline app/export.go:247]: Sometimes it's `panic`, sometimes it's `log.Fatal`. Make it consistent.

File: app/export.go (hunk 1 above)
Comment (janekolszak) [reply app/export.go:179]: I'm copying how it's done elsewhere

**Pull request merged**